client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -297,7 +298,8 @@ const mapStateToProps = ( state ) => {
 		menusUrl: getCustomizerUrl( state, siteId, 'menus' ),
 		isNewlyCreatedSite: isNewSite( state, siteId ),
 		showCustomizer: ! isSiteUsingFullSiteEditing( state, siteId ),
-		hasCustomDomain: getGSuiteSupportedDomains( domains ).length > 0,
+		hasCustomDomain:
+			getGSuiteSupportedDomains( domains ).length > 0 && canUserPurchaseGSuite( state ),
```

Comment:
👍  Let me know if I can do anything to improve that in this PR.

Reply:
I think we're good for now -- this needs a business logic change rather than a refactor! Let's get this shipped and we can take on working out what the business logic ought to be separately.

Reply:
Thanks for verifying, @daledupreez. Feel free to ping me in any follow-ups if I can help with anything.